application-xpoll-ui/src/main/resources/XPoll/XPollSheet.xml

```diff
@@ -180,6 +195,7 @@
                   &lt;input type="checkbox" name="${proposal.hashCode()}"
                       #if ($checked)CHECKED#end #if ($xcontext.user != $user) disabled #end /&gt;
                 &lt;/td&gt;
+                #end
```

Comment:
Wrong indentation

Reply:
This is good, there is a `#foreach` that corresponds to this `#end` which is collapsed by github.